taxonomy-editor-frontend/src/App.js

```diff
@@ -0,0 +1,26 @@
+import { createTheme, CssBaseline, ThemeProvider } from "@mui/material";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Home from './pages/home';
+
+const theme = createTheme({
+  typography: {
+      fontFamily : 'Plus Jakarta Sans',
+  },
+});
+
+function App() {
+  return (
+    <Router>
+      <ThemeProvider theme={theme}>
+        <div className="App">
```

Comment:
is this needed? Also for the order, think that the ThemeProvider and CssBaseline could go outside of the Router